app.js

```diff
@@ -27,4 +27,32 @@ app.get('/', function(req, res) {
     });
 });
 
+app.get('/search', function (req, res) {
+    var keyword = req.query.keyword;
+    if (keyword === undefined) {
```

Comment:
falsy kullanabilirdin, kullanmamanin ozel bir sebebi var mi?

Reply:
ben bu falsy'i bi sevemedim ama seni mi kırcam be.